students/JasneetChandok/Lesson04/fileexcercise.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 4: File Excercise'''
+
+import os
+import glob
+import pathlib
+
+# Path and File Processing
+
+# Part 1
+folder_path = os.getcwd()
+list_of_file = glob.glob(os.path.join(folder_path, '*.py'))
+
+for file_name in list_of_file:
+    print(file_name)
+
+
+# Part 2
+path = pathlib.Path.cwd()
+sourcefile_path = 'C:\\Users\\jchando1\\OneDrive - T-Mobile USA\\Programming in Python\\SP_2019_210A_classroom\\students\\JasneetChandok\\Readme.txt'

```

Comment:
Be careful here, you don't want to hardcode your path. You want to use the os.path to find the path and then you can append "/outfile.txt"